components/ShadowElevations/examples/ShadowElevationsTypicalUseExample.m

```diff
@@ -124,10 +201,65 @@ - (void)viewDidLoad {
   self.view.backgroundColor = [UIColor whiteColor];
   self.title = @"Shadow Elevations";
   _shadowsView = [[ShadowElevationsPointsView alloc] initWithFrame:self.view.bounds];
-  _shadowsView.autoresizingMask =
-      UIViewAutoresizingFlexibleHeight | UIViewAutoresizingFlexibleWidth;
   [self.view addSubview:_shadowsView];
+  
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
```

Comment:
I don't think you need any of these `#if` checks

Reply:
Thanks for the comment! I feel this part is confusing too. What I wanted to do was using the `autoresizingMask` when it is `>= iOS11` and let the `viewSafeAreaInsetsDidChange` to handle `shadowView`'s position. When the OS is below 11, it uses `setupShadowsViewConstraints` to set up the constraint. How do you feel about it?

Reply:
I just realized `#if ` is not needed anymore. I misunderstood it, and please ignore my previous comment. Thanks for pointing it out!